src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -2659,9 +2659,6 @@
   <data name="IO_BindHandleFailed" xml:space="preserve">
     <value>BindHandle for ThreadPool failed on this handle.</value>
   </data>
-  <data name="IO_InconsistentLinkType" xml:space="preserve">
-    <value>The link's file system entry type is inconsistent with that of its target: {0}</value>
-  </data>
```

Comment:
nit: it would be nice if you could just remove `IO_InconsistentLinkType` without modifying the rest of the file (I know that these were just whitespaces but still ;) )

Reply:
These changes were made automatically by VS' resx editor. Yes, I agree they just pollute PRs but I didn't even noticed it this time. Will fix.